chimney/src/main/scala/io/scalaland/chimney/internal/macros/TransformerMacros.scala

```diff
@@ -129,6 +132,8 @@ trait TransformerMacros extends TransformerConfiguration with MappingMacros with
       expandSourceWrappedInOption(srcPrefixTree, config)(From, To)
     } else if (bothEithers(From, To)) {
       expandEithers(srcPrefixTree, config)(From, To)
+    } else if (bothMaps(From, To)) {
```

Comment:
Generalized rule for case when only `From` is map. We can't handle it in `expandIterableOrArray` cause resolveRecursiveTransformerBody for tuple (key, value) and it will be tagged by tuple rule

Reply:
Ok. Current implementation delegating all other cases to `expandIterableOrArray` seems fine too.